app/views/admin/loans/index.html.slim

```diff
@@ -0,0 +1,74 @@
+- content_for :title do
+  - 'Loans'
+
+= render 'layouts/admin/tables'
+
+= define_grid(@loans_grid, show_filters: :when_filtered) do |g|
+
+  - g.blank_slate do
+    - @empty_list = true
+    - content_tag(:div, "No records found", class: 'alert alert-info')
+
+  - g.column name: 'ID', attribute: 'id'
+
+  - g.column name: 'Division', attribute: 'name', assoc: :division,
+    custom_filter: :auto, detach_with_id: :division_filter
+
+  - g.column name: 'Co-op', attribute: 'name', assoc: :organization,
+    custom_filter: :auto, detach_with_id: :coop_filter
```

Comment:
For some reason I am getting a bunch of coops in the coop filter for which there are no loans defined. @linesarefuzzy any thoughts on this?